File: problems/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>


void print_wall(int i);

int main(void)
{

    int height;
    do
    {
        height =  get_int("Height: ");

    }
    while (height > 8 || height < 1);


    for (int i = 0; i < height; i++)
    {

        for (int x = 1; x < height - i; x++)
        {
            printf(" ");
        }

        print_wall(i);

        printf("  ");

        print_wall(i);

        printf("\n");
    }

}

void print_wall(int i)
{
    for (int j = 0; j <= i; j++)
    {
        printf("#");
    }
}

File: problems/sentimental/mario-more/mario.py
from cs50 import get_int
# TODO


while (True):
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(n):

    print(" " * (n-1-i), end="")
    print("#" * (i+1), end="")
    print(" " * 2, end="")
    print("#" * (i+1))

